app/controllers/api/v1/greetings_controller.rb

```diff
@@ -0,0 +1,53 @@
+class Api::V1::GreetingsController < ApplicationController
+  before_action :set_greeting, only: %i[show update destroy]
+
+  # GET /greetings
+  def index
+    @greetings = Greeting.order('RANDOM()').first
+    # Greeting.order("RANDOM()").first
+
+    render json: @greetings
+  end
+
+  # GET /greetings/1
+  def show
+    render json: @greeting
+  end
+
+  # POST /greetings
+  def create
+    @greeting = Greeting.new(greeting_params)
+
+    if @greeting.save
+      render json: @greeting, status: :created, location: @greeting
+    else
+      render json: @greeting.errors, status: :unprocessable_entity
+    end
+  end
```

Comment:
* Your backend API works fine as shown in the screenshot below 👏 
* But your frontend app doesn't follow the [proper requirement ](https://github.com/microverseinc/curriculum-rails/blob/main/connect-frontend-frameworks/hello_world_two_apps.md#:~:text=Create%20a%20store%2C%20an%20action%20and%20a%20reducer%20that%20will%20connect%20to%20you%20API%20endpont%20to%20get%20the%20random%20greeting.)for setup and it doesn't display any message from the API Kindly fix these errors 🙏 

> Create a store, an action, and a reducer that will connect to your API endpoint to get the random greeting. 

![image](https://user-images.githubusercontent.com/52497246/197367797-61e4aa17-baa8-42f3-884f-2d52325bfab3.png)

Reply:
done.
![image](https://user-images.githubusercontent.com/13853450/197368810-84a82b90-0030-46bb-a8e5-f7f0a8c25777.png)